packages/ember-data/lib/serializers/json-api-serializer.js

```diff
@@ -0,0 +1,295 @@
+/**
+  @module ember-data
+*/
+
+import Serializer from "ember-data/system/serializer";
+
+var copy = Ember.copy;
+var get = Ember.get;
+var forEach = Ember.ArrayPolyfills.forEach;
+var map = Ember.ArrayPolyfills.map;
+
+/**
+  @class JSONAPISerializer
+  @namespace DS
+  @extends DS.Serializer
+*/
+export default Serializer.extend({
```

Comment:
Should this extend from `JSONSerializer` so it can inherit the `transformFor` method?